test/trivial/diten/testIntExp4.good

```diff
@@ -1 +1 @@
-error: 0 cannot be raised to a negative power
```

Comment:
Note that the source of this error is a USR_FATAL print inside of compiler/ifa/num.cpp where there are no AST nodes in sight...